site/content/en/docs/contrib/testing.en.md

```diff
@@ -6,17 +6,26 @@ description: >
   How to run tests
 ---
 
-### Unit Tests
+## Prerequisites
+
+* A recent Go distribution (>=1.12)
+* Tools needed for testing :
+
+```shell
+sudo apt-get install libvirt-dev
+```
```

Comment:
Add for debain based

Reply:
Thanks for reviewing :) @medyagh
Since I'm not familiar with other linux distributions, I have a question.
Does your comment mean that Libvirt-dev is unneccessary for other distribution? Or just the install cmd is different?

Reply:
Different command, e.g. `su -c "yum install libvirt-devel"`

Reply:
The root cause for this requirement is that the kvm2 driver is kept within the main source tree (even though the binary is a separate download). If it was a separate git repository, it would make it easier to build minikube.